examples/index_rate_alert.tf

```diff
@@ -0,0 +1,16 @@
+provider "logdna" {
+  servicekey = "Your service key goes here"
+}
+
+resource "logdna_index_rate_alert" "config" {
+  max_lines = 3
+  max_z_score = 3
+  threshold_alert = "separate"
+  frequency = "hourly"
+  enabled = true
+  channels {
+    email = ["test@email.test"]
```

Comment:
test@email.test will result in a 400 response. It is valid response, telling users to use a valid email. However, I assume it's good to use a .com email address so that user can get a positive outcome of 200 response.

Reply:
Fixed